api/envoy/extensions/access_loggers/fluentd/v3/fluentd.proto

```diff
@@ -22,8 +22,42 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // the Fluentd Forward Protocol as described in: `Fluentd Forward Protocol Specification
 // <https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1>`_.
 // [#extension: envoy.access_loggers.fluentd]
-// [#next-free-field: 7]
+// [#next-free-field: 8]
 message FluentdAccessLogConfig {
+  enum BackOffStrategy {
+    // Constructs fully jittered exponential backoff strategy.
+    // The backoff interval value will be in the range of [base_backoff_interval, max_backoff_interval).
+    JitteredExponential = 0;
+
+    // Constructs fully jittered backoff strategy based on the base_backoff_interval value.
+    // The backoff interval value will be in the range of [base_backoff_interval, 1.5 * base_backoff_interval).
+    JitteredLowerBound = 1;
+
+    // Constructs fixed backoff strategy based on the base_backoff_interval value.
+    Fixed = 2;
+  }
+
+  message RetryOptions {
+    // The number of times the logger will attempt to connect to the upstream during reconnects.
+    // By default, there is no limit. The logger will attempt to reconnect to the upstream each time
+    // connecting to the upstream failed or the upstream connection had been closed for any reason.
+    google.protobuf.UInt32Value max_connect_attempts = 1;
+
+    // Sets the backoff strategy. Default value is JitteredExponential.
+    BackOffStrategy backoff_strategy = 2;
+
+    // The base interval to be used for the next back off computation. It should be greater than
+    // 1 millisecond and less than or equal to max_backoff_interval. The default value is 1 millisecond.
+    google.protobuf.Duration base_backoff_interval = 3
+        [(validate.rules).duration = {gte {nanos: 1000000}}];
+
+    // Specifies the maximum interval between retries. This parameter is optional,
+    // but must be greater than or equal to the base_backoff_interval if set. The default
+    // is 10 times the base_backoff_interval. This field only has effect if JitteredExponential backoff
+    // strategy is configured.
+    google.protobuf.Duration max_backoff_interval = 4 [(validate.rules).duration = {gt {}}];
```

Comment:
This might be negated by my either comment to use the common config, but I think the defaults here are too short. This would reconnect every 10ms which is too fast IMO. I would target a default that would end up reconnecting at something > 5s or so once max backoff is reached.

Reply:
I think we can use the common config and comment that the defaults are different for this extension? I remember seeing similar pattern in another extension

Reply:
Sure that seems fine if the defaults really need to change.